features/shared/checkout_previous_branch_after_success/same_current_branch/previous_branch_deleted.feature

```diff
@@ -0,0 +1,39 @@
+Feature: Allow checking out the correct previous Git branch after running a Git Town commmand that leaves the current branch intact and deletes the previous branch
```

Comment:
Same as https://github.com/Originate/git-town/pull/501#discussion_r29312041

So in this case:
> Feature: when the previous branch is deleted during a Git Town command, the main branch becomes the new previous branch